### PR TITLE
Fixed ts-utils import

### PR DESCRIPTION
**Pull request by Dantemss**
For: https://github.com/openstax/unified/issues/2727

Need this because of ESM or something

### CODE REVIEW

**Comment by Dantemss** [comment]
How odd... then the GH action that uses cjs fails... I'll investigate some more

File: jest.config.js [hunk 1]
```diff
@@ -13,7 +13,4 @@ module.exports = {
     "json",
     "node"
   ],
-  moduleNameMapper: {
-    "(.*)@openstax/ts-utils(.*)": "$1@openstax/ts-utils/$2.js"
```
Comment (jivey) [inline jest.config.js:17]: I thought this was doing the same thing as manually adding the extensions to the imports, maybe not?

File: src/components/DropdownMenu.spec.tsx [hunk 2]
```diff
@@ -17,70 +17,75 @@ describe('DropdownMenu', () => {
     </DropdownMenu>
   );
 
-  it('should open on a button click and close on another button click', () => {
-    const component = renderer.create(<TestMenu variant='light'/>);
-    expect(component.toJSON()).toMatchSnapshot();
+  // Need to isolate tests from the 2 libraries in these blocks otherwise we get warnings about using the wrong act()
+  describe('@testing-library/react', () => {
```
Comment (jivey) [inline src/components/DropdownMenu.spec.tsx:21]: Interesting, I wouldn't expect a `describe` to provide more isolation than an `it`, unless using the module name as the describe name is doing something special here (I don't think so?)

**Review by jivey — COMMENTED** [review]
What lead to needing this change? I can probably learn something here regarding how bundling works if you will indulge me :)

File: src/components/DropdownMenu.spec.tsx (hunk 2 above)
Comment (TomWoodward) [reply src/components/DropdownMenu.spec.tsx:21]: yea i'm also not sure how the describe is helping in this case

File: jest.config.js (hunk 1 above)
Comment (Dantemss) [reply jest.config.js:17]: It was fine in the spec files but then in the regular imports that needed the extension outside of jest it started duplicating the extension for me.

**Comment by Dantemss** [comment]
@jivey the change was needed because when I added the ts-utils import for the assertions like Tom asked the tests passed but the code failed to run. Then when I added the .js extension the tests failed.

**Review by jivey — APPROVED** [review]

**Pull request merged**